lib/test_helpers.ml

```diff
@@ -0,0 +1,27 @@
+open Parsetree
+open Typedtree
+open Types
+
+(* Helper library to parse .mli content and return Types.signature *)
+let compile_interface (content : string) : Types.signature =
+  (* Parses and tokenizes the contents of the .mli file*)
+  let parse_interface content =
+    let lexbuf = Lexing.from_string content in
+    try Parse.interface lexbuf
+    with exn ->
+      failwith ("Error while parsing .mli content: " ^ Printexc.to_string exn)
+  in
+  (* Create an env and generate a typedtree representation from the parsed interface *)
+  let typedtree_of_interface intf =
+    let initial_env = Compmisc.initial_env () in
+    let env =
```

Comment:
Why are you defining this `env` since you already have one from `Compmisc.initial_env ()`? The `initial_env` you defined using `Compmisc` is probably closer to what we want to use as it is properly initialized and has the stdlib in scope.

I think you can remove this one.